pkg/controller/apiserver/apiserver_controller.go

```diff
@@ -408,6 +418,10 @@ func (r *ReconcileAPIServer) Reconcile(ctx context.Context, request reconcile.Re
 		return reconcile.Result{}, err
 	}
 
+	// Check if the legacy namespace 'tigera-system' exists and can be cleaned up
+	canCleanupOlderResources := false
```

Comment:
Looks like line 422 can be removed. Optionally, 423 can be removed and called in-line @446.